PowerUp/app/build.gradle

```diff
@@ -22,12 +21,14 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    buildToolsVersion '27.0.3'
```

Comment:
Declare this in the project level build.gradle file and use it here as a variable